spec/models/flight_detail_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FlightDetail, type: :model do
+  it { should have_many(:related_connections) }
```

Comment:
Uma vez que a [documentação do RSpec](https://github.com/rspec/rspec-expectations/blob/main/Should.md) sugere, há 6 anos, substituir `should` por `expect`, que tal usar a sintaxe mais recomendada em testes como este?

```suggestion
  it { is_expected.to have_many(:related_connections) }
```

Reply:
Ajustado em: https://github.com/Laryssa-lfa/flight-company/pull/5/commits/06ba234eb9f57962438ab44d70c9219543cf58f0.